Documentation/gettingstarted.rst

```diff
@@ -1042,39 +1040,37 @@ You can observe the newly created endpoints in Cilium, similar to the following
     33115      Disabled      257        mesos:id=goodclient           f00d::a00:20f:0:815b   10.15.220.6    ready
     64189      Disabled      258        mesos:id=badclient            f00d::a00:20f:0:fabd   10.15.152.27   ready    
 
-Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. Tail the *goodclient* logs:
+Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. In a new terminal, go to ``/cilium/examples/mesos``, start a new ssh session to the Vagrant VM with ``vagrant ssh`` and tail the *goodclient* logs:
 
 ::
 
     $ ./tail_client.sh goodclient
 
-and in a separate terminal, start another ssh session to the Vagrant VM with ``vagrant ssh`` and observe the *badclient* logs:
+and in a separate terminal, do the same thing with ``vagrant ssh`` and observe the *badclient* logs:
 
 ::
 
     $ ./tail_client.sh badclient
 
-Make sure the previous command continuously prints the result of the client accessing */public* and */private* API of the web-server:
+Make sure both tail logs continuously prints the result of the clients accessing the */public* and */private* API of the web-server:
 
 ::
 
-    ...
-    --------------------------------
-    GET /public
-    OK
-    
-    HTTP code: 200
-    GET /private
-    OK
-    
-    HTTP code: 200
-    --------------------------------
-    ...
+     ...
+     ---------- Test #X  ----------
+        Request:   GET /public
+        Reply:     OK
+      
+        Request:   GET /private
+        Reply:     OK
+     -------------------------------
```

Comment:
I like the new output 👍

Reply:
Yes, very nice.  After the text "Make sure both tail logs continuously prints the result of the clients accessing the /public and /private API of the web-server"  I might add "Since no Cilium policy has been applied, both clients are able to access the webserver and retrieve both URLs".  Effectively, you want to establish the baseline for the policy tests later.

Reply:
Added.